pkgs/development/python-modules/altgraph/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "altgraph";
+  version = "0.17.3";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ad33358114df7c9416cdb8fa1eaa5852166c505118717021c6a8c7c7abbd03dd";
+  };
+
```

```suggestion

  pythonImportsCheck = [ "altgraph" ]

```
One last thing, you should add otherwise this is good to go. It just tries to import altgraph.